app/views/main/index.html.erb

```diff
@@ -0,0 +1,3 @@
+<h1>This is you login page</h1>
```

Comment:
I think you meant to say "your" :-)

Reply:
You are absolutely right! I don't know why but ever since starting Turing I can't spell!!